utils/zerovec/src/map/hashmap.rs

```diff
@@ -0,0 +1,270 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use super::*;
+use crate::ule::AsULE;
+use crate::ZeroVec;
+use ahash::AHasher;
+use alloc::borrow::Borrow;
+use alloc::vec::{from_elem, Vec};
+use core::hash::{Hash, Hasher};
+
+#[inline]
+fn create_hasher_with_seed(seed: u128) -> AHasher {
+    AHasher::new_with_keys(seed, 0xaabbccdd)
+}
+
+#[inline]
+fn compute_hash<K: Hash>(seed: u32, k: K, m: usize) -> u32 {
```

Comment:
In `Index` the result of `compute_hash` is always cast to `usize`. Why is it returning `u32`?

Reply:
Yes realized this, changed the return type to usize.